pages/index.tsx

```diff
@@ -264,8 +266,10 @@ const Home = ({ featuredTopicRef }: { featuredTopicRef: string }) => {
         {/* Beneath the Surface initiative - About snippet */}
         <div className="w-100 bg-grey-7 p-3 p-md-5 d-flex flex-column align-items-center">
           <div className="d-flex gap-3 align-items-center justify-content-center w-100 mt-2">
-            <Image alt="BTS Crane" src="/photos/home/hp_crane.png" className="home-logo-crane" />
-            <Image alt="BTS Logo" src="/photos/home/hp_logo.png" className="home-logo-text" />
+            {i18n.language === "en" &&
+              <Image alt="BTS Crane" src={`/photos/home/hp_crane_en.png`} className="home-logo-crane" />
+            }
+            <Image alt="BTS Logo" src={`/photos/home/hp_logo_${i18n.language}.png`} className="home-logo-text" />
```

Comment:
could we not just rename hp_crane_en,png to hp_logo_en? or is there a specific reason?

Reply:
Sorry this was a bit of a jank solution! Long story short, that little segment on the EN version is actually two images (because one image in that specific format was not available), but for JP its one image. I don't think this will be like this in the long term, and I'm not sure if Lydia will have that specific english image available by the conference, not sure if its worth pushing her on this or not